calculate_age/calculate.py

```diff
@@ -0,0 +1,69 @@
+import time
+from calendar import isleap
+
+# judge the leap year
+def judge_leap_year(year):
```

Comment:
We don't need to create a new method because this method existed in Python Library. Don't try to create a method which existed in core library.